Controllers/PersonController.cs

```diff
@@ -24,24 +21,143 @@ public PersonController(ApplicationDbContext context)
             _context = context;
         }
 
-        // GET: /Person/Index/
-        public async Task<IActionResult> IndexAsync()
+        // GET: Person
+        public IActionResult Index()
         {
-            List<Person> persons = await _context.Persons.ToListAsync<Person>();
-            List<PersonIndexViewModel> viewModels = persons.ConvertAll<PersonIndexViewModel>(person => new PersonIndexViewModel(person));
-            viewModels.Sort((x, y) => x.FirstName.CompareTo(y.FirstName));
-            return View(viewModels);
+            var persons = _context.Persons.
+                Include(x => x.PersonsMemberStatus).
+                ThenInclude(x => x.MemberStatus).
+                Include(x => x.PersonsCareerLevels).
+                ThenInclude(x => x.CareerLevel).
+                Include(x => x.PersonsPositions).
+                ThenInclude(x => x.Position).
+                OrderBy(x => x.LastName).
+                ThenBy(x => x.FirstName);
+
+            return View(persons);
         }
 
-        // Partial view
-        // GET: /Person/_PersonDetailsPartial/
-        public async Task<IActionResult> PersonDetailsPartial(int id)
+        // GET: Person/Details/5
+        public async Task<IActionResult> Details(int? id)
         {
+            if (id == null)
+            {
+                return NotFound();
+            }
             var persons = _context.Persons.Include(x => x.PersonsMemberStatus).ThenInclude(x => x.MemberStatus).Include(x => x.PersonsCareerLevels).ThenInclude(x => x.CareerLevel).Include(x => x.PersonsPositions).ThenInclude(x => x.Position);
-            Person person = await persons.FirstOrDefaultAsync(x => x.PersonID == id);
-            PersonDetailsViewModel viewModel = new PersonDetailsViewModel(person);
+            var person = await persons.FirstOrDefaultAsync(m => m.PersonID == id);
+            if (person == null)
+            {
+                return NotFound();
+            }
+            return View(new PersonDetailsViewModel(person));
+        }
 
-            return PartialView("_PersonDetailsPartial", viewModel);
+        // GET: Person/Create
+        public IActionResult Create()
+        {
+            return View();
+        }
```

Comment:
Is there a link that leads to this action? I couldn't find one, but there should be one added to the Person list